app/assets/javascripts/admin/addon/components/admin-search.gjs

```diff
@@ -91,7 +101,9 @@ export default class AdminSearch extends Component {
   changeSearchTerm(event) {
     this.searchResults = [];
     this.filter = event.target.value;
-    this.runSearch();
+    if (this.filter.length > 0) {
+      this.runSearch();
```

Comment:
Not directly related to this PR but I don't think we are debouncing search at all.